packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -4962,6 +4962,9 @@ class RenderFollowerLayer extends RenderProxyBox {
 
   @override
   bool hitTest(BoxHitTestResult result, { Offset position }) {
+    // Disables the hit testing if this render object is hidden.
+    if (link.leader == null && !showWhenUnlinked)
```

Comment:
Can you also update the documentation on showWhenUnlinked to indicate that it also affects hit testing.